cluster/gce/config-test.sh

```diff
@@ -25,6 +25,7 @@ MASTER_DISK_TYPE=pd-ssd
 MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-20GB}
 NODE_DISK_TYPE=${NODE_DISK_TYPE:-pd-standard}
 NODE_DISK_SIZE=${NODE_DISK_SIZE:-100GB}
+KUBELET_PORT=10250
```

Comment:
Why add this line? We can override this var if we want, but (except for the hollow kube tests) there shouldn't be any reason to change it. Also note that by only changing it here and not in config-default.sh we are potentially breaking cluster-up for non-e2e use cases without realizing it.